src/features/MoviesCards/MoviesCards.jsx

```diff
@@ -21,56 +16,17 @@ function MoviesCards() {
       <div className="d-flex">
         <div className="col-12">
           <div className="row mb-5 mt-2">
-            <div className="col-lg-4 col-lg-3 col-md-6 col-sm-12 col-xs-12 px-5">
+            {data.map((item) => (
               <MoviesCard
-                src={card1}
-                name="Pulp Fiction"
-                type="Action & Adventure"
-                year="2004"
+                key={item.id}
+                item={item}
+                src={item.src}
+                name={item.name}
+                type={item.type}
+                year={item.year}
+                text={item.text}
```

Comment:
good approach! some students are tempted to use {...item}, but passing parameters explicitly gives more visability that we're not passing something we don't need

Although, if you're already passing the whole `item={item}`, you don't need to additionally pass other parameters that are already inside :) you're duplicating data you're passing and creating 2 sources of truth

Reply:
Wow, well I wrote this code without knowing😃 it just came from intuition. I felt like I don't need a whole of propersties of item like I should specify the exact props so i wrote like this